### ISSUE TITLE
Logged out metadata on posts and profiles is currently broken

### ISSUE THREAD

**Issue by Jonty** [bluesky-social/social-app#975]
**Describe the bug**

Logged out metadata about posts and profiles is currently broken.

The main page tags exist but are empty. The meta tags are missing due to [postView](https://github.com/bluesky-social/social-app/blob/49840f3a2739ea0c6134f1801dd843ec4771bccb/bskyweb/cmd/bskyweb/server.go#L289C4-L289C4) being null.

**To Reproduce**

```
curl https://bsky.app/profile/bsky.app/post/3jzpikdls2z23 | grep bsky_
    <p id="bsky_display_name"></p>
    <p id="bsky_handle"></p>
    <p id="bsky_did"></p>
    <p id="bsky_post_text"></p>
```

```
curl https://bsky.app/profile/bsky.app | grep bsky_
    <p id="bsky_display_name"></p>
    <p id="bsky_handle"></p>
    <p id="bsky_did"></p>
    <p id="bsky_profile_description"></p>
```

**Expected behavior**

The tags are populated, and the [```<meta>``` tags](https://github.com/bluesky-social/social-app/blob/49840f3a2739ea0c6134f1801dd843ec4771bccb/bskyweb/templates/post.html#L12-L38) are added to the page and populated.

**Additional context**

I suspect this may be due to the Go server in production having expired API credentials, so is unable to fetch the post/profile data.

**Comment by pfrazee** [comment]
@Jacob2161 can you double check this for me? <3

**Comment by Jonty** [comment]
And, it's now broken again! 😭

**Comment by Jacob2161** [comment]
Yeah, this is broken and always has been I believe. I have a fix to refresh the token and will try to test it out and get it up today.

**Comment by Jonty** [comment]
This was fixed by #1065

**Comment by Jonty** [comment]
Well, it's broken again! I can't see an obvious code change that has broken it though.

**Comment by mozzius** [comment]
Is this still broken?

**Comment by mozzius** [comment]
Cannot repro - will close

<img width="668" height="356" alt="Image" src="https://github.com/user-attachments/assets/65bce8e9-46dd-443a-83c6-1bdc3e42ba10" />
